sci-biology/spm/ChangeLog

```diff
@@ -0,0 +1,7 @@
+# ChangeLog for sci-biology/spm
+# Copyright 1999-2015 Gentoo Foundation; Distributed under the GPL v2
+# $Header: $
+
+  17 Feb 2015;  <chymera@gentoo.org> +metadata.xml, +spm-8.ebuild.orig,
```

Comment:
`repoman` is calling `echangelog` (see `man repoman` for details) and `echangelog` uses the perl function `getpwuid`, which uses the account information from your system (see `man echangelog` for details). The `ECHANGELOG_USER` variable is just the easiest way to overwrite your, maybe wrong, system settings. There is basically no harm in adding it to your `~/.bashrc`, but you could also export it and later unset it again.

And yeah, `echangelog` could use the information for your git settings, but @agriffis has implemented that back in 2009.